dbx/jvm/v5.6-wn-items.rst

```diff
@@ -1,12 +1,55 @@
-- Adds two new SDAM error messages for when the server description changes to
-  ``UNKNOWN`` due to a stale primary node. To learn more about these error
-  messages, see the :spec:`Server Discovery and Monitoring
-  </server-discovery-and-monitoring/server-discovery-and-monitoring.md#error>`
-  Specification in GitHub.
+- Adds experimental support for Queryable Encryption prefix, suffix, and substring queries.
+  To learn more about QE, see :ref:`In-Use Encryption <csharp-fle>`.
```

Comment:
This link looks wrong - links to csharp-fle?

Reply:
Good catch!